streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -138,6 +138,7 @@
 
     /** {@code client.id} */
     public static final String CLIENT_ID_CONFIG = CommonClientConfigs.CLIENT_ID_CONFIG;
+    private static final String CLIENT_ID_DOC = "An id string, which will be used as a prefix for internal consumer, producer and restore-consumer";
```

Comment:
Hi mjsax, Thanks for the comment. Does following message sounds good? 

"An id string used by internal clients to pass to server when making requests. It will be passed in kafka protocol request header. The format of that header is {api_key:api_version:correlation_id:client_id} "